src/app/prevention/strides-against/game-config.ts

```diff
@@ -0,0 +1,138 @@
+import { Item } from './models/Item';
+import { BackgroundElement, ManElement, PositionElement } from './models/elements';
+
+export const BackgroundConfig: BackgroundElement = {
+  imageURL: '/assets/img/strides-against/background.jpg',
+  imageX: 0,
+  imageY: 0,
+  imageWidth: 800,
+  imageHeight: 450,
+};
+
+export const ManConfig: ManElement = {
+  imageURL: '/assets/img/strides-against/man.svg',
+  imageX: 90,
+  imageY: 92,
+  imageWidth: 25,
+  imageHeight: 50,
+};
+export const ObjectsConfig: Array<Item> = [
+  {
+    name: 'Garlic',
+    index: 0,
+    element: {
+      imageURL: '/assets/img/strides-against/garlic.png',
+      messageImageURL: '/assets/img/strides-against/messages/garlic.jpg',
+      messageBody: 'It\'s just a myth, consuming Garlic to prevent Malaria does not work.',
```

Comment:
The english sentences can be fetched from english.json right?

Reply:
I don't know but I may face difficulty in subscribing to Language service in a normal TS file given we don't have lifecycle hook in it. If language is a problem, we can have element objects in it (i.e. config file) for different languages and call from the component for a specific language. That would also help in keeping things related to it in its vicinity.

Reply:
I had the same comment .. it's somehow gone now.. is it due to reading?

Reply:
It's available in "outdated" label.

Reply:
Hmmm.... This is a problem. I think we have to either follow your suggestion which I think will require changes in multiple places whenever we add a new language or move objectsconfig to the component which I think will not be a clean way of implementation.